### DISCUSSION TITLE
기술, Kit 사용 개발 Tip 공유해보아요!

### DISCUSSION THREAD

**Discussion by manju-minji** [DeveloperAcademy-POSTECH/.github#74] [General]
특정 Kit, 기술들을 사용하다가 다른 사람들이 해당 기술을 사용할 때 참고했으면 하는 정보들, 내가 사용한 Tip이 있나요?
있다면 자유롭게 공유해주세요!!

## CarPlay
고려할 점: 개발을 위해 apple로부터의 entitlements를 요청 받아야 하지만, 이 과정이 길고/쉽지 않다.

## Apple Pay
고려할 점: PG사 등록이 필요하다.

## CloudKit, StoreKit, WeatherKit, MusicKit
고려할 점: 여러 명이서 작업하기에 팀 계정이 아닌 경우에는 어렵다.

**Comment by yeniful** [comment]
NC2 때의 기억을 떠올리면 도움이 많이 되겠군요 후후..!!!
